05.wc/wcc.rb

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+WIDTH_SUM_VALUE = 8
```

Comment:
SUM=合計、というのは何の合計を意図していますか？
あとVALUEというのはDATAとかと同じで「無味無臭な名前」なのであえて付けなくてもいいですね。
個人的には幅の大きさを表す定数だよ、という意味で、

```suggestion
WIDTH = 8
```

でも十分かな〜と思います。

Reply:
承知しました。ご指摘のとおり修正しました